modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -326,7 +326,7 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
           const rng = dom.createRng();
           rng.setStartBefore(node);
           rng.setEndAfter(node);
-          applyRngStyle(dom, ExpandRange.expandRng(ed, rng, formatList), true);
+          applyRngStyle(dom, ExpandRange.expandRng(ed.dom, rng, formatList), true);
```

Comment:
```suggestion
          applyRngStyle(dom, ExpandRange.expandRng(dom, rng, formatList), true);
```
`dom` is already in scope here so we don't need to do this.